bitnami/redis/templates/configmap-scripts.yaml

```diff
@@ -23,14 +23,16 @@ data:
         fi
     }
 
+    NODES_ALIVE="$(getent hosts "{{ template "redis.fullname" . }}-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}")"
+
     export REDIS_REPLICATION_MODE="slave"
     if [[ $HOSTNAME =~ (.*)-([0-9]+)$ ]]; then
-      if [[ ${BASH_REMATCH[2]} == "0" ]]; then
```

Comment:
I guess you don't need to compare the `$HOSTNAME` variable if you're not using its result.

Reply:
Absolutely. I guess it remained either because of keeping the diff small(er) or because the initial attempt was to do minimal change. But I will remove the unnecessary layer of if.